### ISSUE TITLE
E2EU crash when attempting to add SAG Mill recipe to Pattern Terminal through HEI if recipe has a ginding ball. 

### ISSUE THREAD

**Issue by ghost** [CleanroomMC/HadEnoughItems#80]
After a second or two (probably from the auto-cycling through the grinding balls) after selecting the SAG Mill to look at the recipe for the Flux Core, with plans to use the + recipe button for the ME pattern terminal, the game crashes. Happened twice, here are the logs, which are almost exactly the same from my quick glance:

https://mclo.gs/3bqqI6Q
https://mclo.gs/o0wqDvN

Got another one, this time it was when I hovered over the + button to add recipe to the pattern terminal. Perhaps this was the original cause.
https://mclo.gs/zrYSEhZ

Tried again, this time with a different recipe that has a grinding ball in the recipe, crashed when hovering over the +. Doesn't happen on recipes without a grinding ball.
https://mclo.gs/4uIahOe

I have posted this to EnderIO as well.
https://github.com/Team-EnderIO/EnderIO/issues/763

**Comment by WaitingIdly** [comment]
this is the fault of Not Enough Energistics https://www.curseforge.com/minecraft/mc-mods/not-enough-energistics
and is clearly shown by the stacktrace here:
```
    at com.github.vfyjxf.nee.helper.RecipeAnalyzer.lambda$analyzeRecipe$2(RecipeAnalyzer.java:110)
    at com.github.vfyjxf.nee.helper.RecipeAnalyzer$$Lambda$7843/715139726.test(Unknown Source)
```
this also happens with JEI + NEE, so it is not the cause of HEI.
this has already been reported to NEE here https://github.com/vfyjxf/NotEnoughEnergistics/issues/39 and resolved - you are playing with `2.0.3` and the most recent version is `2.0.7`.

**Comment by ghost** [comment]
Ah, thank you. E2EU didn't show me any mods that needed to be updated. I did this one manually now.
